projects/canopy/src/lib/modal/modal.component.ts

```diff
@@ -69,7 +69,7 @@ export class LgModalComponent implements OnInit, AfterContentInit, OnDestroy {
           if (isOpen) {
             bodyEl.style.overflow = 'hidden';
             this.open.emit();
-          } else {
+          } else if (isOpen !== undefined) {
             this.closed.emit();
             bodyEl.style.overflow = '';
           }
```

Comment:
should there be an else here that sets some default state/behaviour?

Reply:
No, it will simply do nothin gif it's the first emission 🤷‍♀️